google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -182,6 +182,7 @@
     <project.converterProvider implementation="com.google.cloud.tools.intellij.appengine.migration.AppEngineStandardFacetConverterProvider"/>
     <project.converterProvider implementation="com.google.cloud.tools.intellij.appengine.migration.AppEngineStandardFacetMigrationConverterProvider"/>
     <framework.detector implementation="com.google.cloud.tools.intellij.appengine.facet.AppEngineFrameworkDetector"/>
+    <compileServer.plugin classpath="google-cloud-tools-jps-plugin.jar;appengine-plugins-core-0.2.0.jar;"/>
```

Comment:
@aslo I think the build plugins use different class loaders to IJ, so they're much lighter to load and faster. This is how stuff is added to the build plugin classpath. Still at a miss as to how script the version number here...